nilearn/datasets.py

```diff
@@ -26,6 +25,15 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    _get_time = uptime.uptime
```

Comment:
Indeed, according to the docs, there *IS* no error thrown--`uptime.uptime()` simply returns `None`.   Will account for that...